en/lessons/libraries/nimble-publisher.md

```diff
@@ -49,6 +55,20 @@ A typical blog post will be written in Markdown syntax, with a metadata section
 Yes, this is **the post** you've been waiting for.
 ```
 
+and add these content for the `/priv/posts/2020/11-04-exciting-news.md` file:
```

Comment:
Too low level as well. We don't need to instruct the reader how to do these operations.